src/styles.css

```diff
@@ -80,6 +131,7 @@ button {
 }
 
 .nav-bar__nav-tabs > button {
+  border-bottom: 4px solid transparent;
```

Comment:
this looks great! if we could also add a transition that goes from 0px to 4px if we want to see the border grow vs just having it appear - this looks great though.

Reply:
YES, let's try to do that together. I attempted it but was not successful (i.e., the text in the button would disappear with the line).